client/my-sites/site-settings/main.jsx

```diff
@@ -25,31 +24,21 @@ import SiteSettingsNavigation from './navigation';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import JetpackDevModeNotice from './jetpack-dev-mode-notice';
 
-/**
- * Module vars
- */
-const debug = debugFactory( 'calypso:my-sites:site-settings' );
-
 export class SiteSettingsComponent extends Component {
-	constructor( props ) {
-		super( props );
-
-		// bound methods
-		this.updateSite = this.updateSite.bind( this );
-
-		this.state = {
-			site: this.props.sites.getSelectedSite()
-		};
-	}
 
-	componentWillMount() {
-		debug( 'Mounting SiteSettings React component.' );
-		this.props.sites.on( 'change', this.updateSite );
-	}
+	static propTypes = {
+		hasLoadedSitePurchasesFromServer: PropTypes.bool.isRequired,
```

Comment:
Can we separate the connected props from the rest of them? Something I've seen being done recently:

```
siteId: PropTypes.number,
// Connected props
isRequesting: PropTypes.bool.isRequired,
```